modules/test/poshi/poshi-runner/src/main/java/com/liferay/poshi/runner/selenium/BaseWebDriverImpl.java

```diff
@@ -1419,6 +1434,10 @@ public String getSikuliImagesDirName() {
 		return _sikuliImagesDirName;
```

Comment:
@lucasneves91 it's missing return in the _ocularSnapImageDirName and _ocularResultImageDirName ?

Reply:
No, it's not! See lines 1437~1440 below.